### PR TITLE
Updated the bundle install and uninstall articles for Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: bundles/installation.rst [hunk 1]
```diff
@@ -48,89 +55,56 @@ version, include it as the second argument of the `composer require`_ command:
...
-Instead of the full bundle name, you can also pass the short name used as the root
-of the bundle's configuration:
+Otherwise, you need the create those configuration files manually following the
+bundle's documentation. In case you need it, you can also get a reference of the
+bundle's configuration via the ``config:dump-reference`` command:
 
 .. code-block:: terminal
 
-    $ bin/console config:dump-reference assetic
+    $ bin/console config:dump-reference WebProfilerBundle
```
Comment (wouterj) [inline bundles/installation.rst]: `php bin/console config:dump-reference WebProfilerBundle`

File: bundles/installation.rst [hunk 2]
```diff
@@ -48,89 +55,56 @@ version, include it as the second argument of the `composer require`_ command:
...
+
+    # you can also pass the bundle alias used in its config files:
+    # bin/console config:dump-reference web_profiler
```
Comment (wouterj) [inline bundles/installation.rst]: `$ php bin/console ...`

File: bundles/installation.rst [hunk 3]
```diff
@@ -140,21 +114,23 @@ The output will look like this:
 
     .. code-block:: terminal
 
-        $ bin/console config:dump-reference AsseticBundle use_controller
+        $ bin/console config:dump-reference TwigBundle date
```
Comment (wouterj) [inline bundles/installation.rst]: `php bin/console`

File: bundles/remove.rst [hunk 4]
```diff
@@ -4,91 +4,53 @@
...
+  ``config/routes/fos_js_routing.yaml`` for FOSJsRoutingBundle).
 
-.. tip::
+2. Remove Bundle Assets
+-----------------------
+
+If the bundle included web assets, remove them from the ``public/bundles/``
+directory (e.g. ``public/bundles/nelmioapidoc/`` for the NelmioApiDocBundle).
+
+3. Remove Integration in other Bundles
```
Comment (wouterj) [inline bundles/remove.rst]: whoa, we shouldn't edit third party bundles to remove their integration? I know you don't mean that, but that title seems to indicate it (and I'm also not very sure what you do mean with this section).

As soon as bundle A depends on B, composer will not allow you to remove B so I don't think there is a problem here?

What we can say is that you have to remove the integration in your application. But that's rather obvious...

**Review by wouterj — CHANGES_REQUESTED** [review]

File: bundles/remove.rst (hunk 4 above)
Comment (javiereguiluz) [reply bundles/remove.rst]: I didn't change that from the original article. See https://symfony.com/doc/current/bundles/remove.html#remove-integration-in-other-bundles

**Comment by weaverryan** [comment]
In #8681 I've also proposed removing the `remove.rst` article entirely

**Pull request closed without merging**